src/app/exam/questions/ExamQuestions.js

```diff
@@ -101,58 +102,52 @@ const ExamQuestions = () => {
                     width="1200px"
                     title="Questions"
                     tableHeaders={[
-                        <TableCell>#</TableCell>,
-                        <TableCell>Question ID</TableCell>,
-                        <TableCell>Question Title</TableCell>,
-                        <TableCell>Score Percentage</TableCell>,
-                        <TableCell>Submission Quota</TableCell>,
-                        " "]}
-                    tableDataStyle={{height: "60px"}}
+                        "#",
+                        "Question",
+                        <p className="text-center">Score Percentage</p>,
+                        <p className="text-center">Submission Quota</p>,
+                        " "
+                    ]}
+                    tableDataStyle={{height: "24px", padding: "5px 12px"}}
                     tableRowGenerator={{
                         list: questions,
                         key: question => `${question.questionOrder}-${question.problemId}`,
-                        data: (question) => {
-                            return [
-                                <TableCell>{toCharacterIndex(question.questionOrder)}</TableCell>,
-                                <FakeLink>{question.problemId}</FakeLink>,
-                                <FakeLink>{question.problemTitle}</FakeLink>,
-                                <TableCell>{question.maxScore}</TableCell>,
-                                <TableCell>{question.quota}</TableCell>,
-                                <TableCell>
-                                    {rejudgingProblemId === question.problemId ?
-                                        <span className="tag"
-                                              style={{
-                                                  backgroundColor: "#FFBB00",
-                                                  color: "white",
-                                                  width: "75px"
-                                              }}>
+                        data: (question) => [
+                            <p>{toCharacterIndex(questions.findIndex(_question => _question.questionOrder === question.questionOrder))}</p>,
+                            <FakeLink>{`${question.problemId} ${question.problemTitle}`}</FakeLink>,
```

Comment:
Can you change this to the real link?
When clicking the questions, it should redirect to the related problem editor.